tools/bin/go_core_race_tests

```diff
@@ -1,23 +1,27 @@
 #!/usr/bin/env bash
 set -ex
 OUTPUT_FILE=${OUTPUT_FILE:-"./output.txt"}
-USE_TEE="${USE_TEE:-true}"
 TIMEOUT="${TIMEOUT:-30s}"
 COUNT="${COUNT:-10}"
 
-# To allow reuse in CI from other repositories
-TOOLS_PATH=${TOOLS_PATH:-"./tools"}
-
-GO_LDFLAGS=$(bash ${TOOLS_PATH}/bin/ldflags)
-use_tee() {
-  if [ "$USE_TEE" = "true" ]; then
-    tee "$@"
+echo "Failed tests and panics: ---------------------"
+echo ""
+if [[ $GITHUB_EVENT_NAME == "schedule" ]]; then
+  if [[  $DEBUG == "true" ]]; then 
+    GORACE="log_path=$PWD/race" go test -json -vet=off -race -shuffle on -timeout "$TIMEOUT" -count "$COUNT" $1 | tee $OUTPUT_FILE
```

Comment:
I think it'd be worth making a `tools/bin/shared` file of some sort that exports a function which toggles between `tee` and `cat` based on `DEBUG`. Then use that func across all of these test scripts.

Reply:
I went with the explicitly clear (albeit verbose) logic on purpose.

We had a `use_tee` method but it was kind of complicated. I also tried another approach using an `eval` statement, but it had some weird behaviour.

Reply:
If we eliminated the requirement of dumping out 300MB+ of log noise via -json/-v, then we could just tee every time inline.